src/hotspot/share/prims/vectorSupport.hpp

```diff
@@ -55,6 +55,8 @@ class VectorSupport : AllStatic {
     VECTOR_OP_NEG     = 1,
     VECTOR_OP_SQRT    = 2,
     VECTOR_OP_BIT_COUNT = 3,
+    VECTOR_OP_CTZ     = 29,
+    VECTOR_OP_CLZ     = 30,
```

Comment:
Seems weird, could you please either move these two to the end of this enum or set the number to "4, 5"?